test/util/bspNet/fileHelpers.ts

```diff
@@ -13,16 +13,19 @@ export const sendNewStorageRequest = async (
   source: string,
   location: string,
   bucketName: string,
-  valuePropId?: HexString,
-  mspId?: HexString,
+  valuePropId?: HexString | null,
+  mspId?: HexString | null,
```

Comment:
These parameters are optional (can be `undefined`) but you also make it possible to send `null`. It's a bit confusing.

Reply:
We are using `undefined` in many places as "this has not been provided, I will retrieve a default value (first value prop of an MSP)" while in this case `null` is an actual "value" and I wanted to keep the same behaviour.

Reply:
Fair point